### PR TITLE
feat: UI/UX multi account

### PR DESCRIPTION
**Pull request by clegirar**
- New gRPC method `UpdateAccount` that update `accountMetadata` in `bertyaccount`
- New UI/UX of multi-account (last screen of this issue https://github.com/berty/berty/issues/2790)
- Fix import account feature
- Fix handle error stream for messenger

This PR fix some UI points :
- Swipe right in home shouldn't trigger anything (ty to @cerisier)

⚠️ WARNING ⚠️
I have delete `embedTor` tag because it cause a crash (WIP)
There is some other crash, on close of the ipfs node (WIP)
Avatars on other accounts can't be loaded (because for an account `1` we can't have the keys of account `2` and `3` to resolve `mediaPrepare` protocol gRPC method)

Signed-off-by: clegirar <clemntgirard@gmail.com>

### CODE REVIEW

File: go/pkg/bertyaccount/service_account.go [hunk 1]
```diff
@@ -441,6 +442,7 @@ func (s *service) createAccountMetadata(accountID string, name string) (*Account
 	}
 
 	meta.LastOpened = time.Now().UnixNano() / 1000
+	meta.CreationDate = time.Now().UnixNano() / 1000
```
Comment (moul) [inline go/pkg/bertyaccount/service_account.go:445]:
```suggestion
	meta.CreationDate = meta.LastOpened
```

File: go/pkg/bertyaccount/service_account.go [hunk 2]
```diff
@@ -457,7 +459,7 @@ func (s *service) createAccountMetadata(accountID string, name string) (*Account
 	return meta, nil
 }
 
-func (s *service) ImportAccount(_ context.Context, req *ImportAccount_Request) (*ImportAccount_Reply, error) {
+func (s *service) ImportAccount(context context.Context, req *ImportAccount_Request) (*ImportAccount_Reply, error) {
```
Comment (moul) [inline go/pkg/bertyaccount/service_account.go:462]:
```suggestion
func (s *service) ImportAccount(ctx context.Context, req *ImportAccount_Request) (*ImportAccount_Reply, error) {
```

(there are specific rules in go when using `context.Context`)

File: go/pkg/bertyaccount/service_account.go [hunk 3]
```diff
@@ -483,6 +485,19 @@ func (s *service) ImportAccount(_ context.Context, req *ImportAccount_Request) (
 		return nil, err
 	}
 
+	p, _ := s.initManager.GetMessengerClient()
+	a, _ := p.AccountGet(context, &messengertypes.AccountGet_Request{})
```
Comment (moul) [inline go/pkg/bertyaccount/service_account.go:489]: you should check errs here

File: api/bertyaccount.proto [hunk 4]
```diff
@@ -76,7 +79,10 @@ message CloseAccountWithProgress {
 message AccountMetadata {
   string account_id = 1 [(gogoproto.customname) = "AccountID"];
   string name = 2;
-  int64 last_opened = 3;
+  string avatar_cid = 3;
```
Comment (moul) [inline api/bertyaccount.proto:82]:
```suggestion
  string avatar_cid = 3 [(gogoproto.customname) = "AvatarCID"];
```

**Comment by zxxma** [comment]
Still need to be improved, but need to be merged, LGTM 👍

**Review by glouvigny — APPROVED** [review]

**Pull request merged**